easybuild/easyconfigs/l/LZO/LZO-2.10-foss-2017a.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::     Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     MIT/GPL
+#
+# Notes::       Adopted from EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#               Fotis Georgatos <fotis@cern.ch>
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'LZO'
+version = '2.10'
+
+homepage = 'http://www.oberhumer.com/opensource/lzo/'
+description = "Portable lossless data compression library"
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [homepage + 'download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072']
+
+toolchainopts = {'pic': True}
```

Comment:
@timeu Last minor remark: please keep `toolchain` & `toolchainopts` together (so move this line up), sorry for the confusion :)